doc/source/dos-script.rst

```diff
@@ -20,6 +20,6 @@ Command Line Options
 --------------------
 
 .. argparse::
-   :filename: ../../scripts/dos.py
+   :module: euphonic.cli.dos
```

Comment:
References to `dos.py` in this file also need to be updated to `euphonic-dos`

Reply:
done